client/src/components/overview/GalleryView.jsx

```diff
@@ -81,7 +60,9 @@ function findAverageRating(ratings)  {
 
 
 // This is the actual functional component
-function Overview(props) {
+function Overview({ currentStyleId, setCurrentStyleId }) {
```

Comment:
What does 'sku' mean?

Reply:
That is the technical term for a product id. But I should change it to match the api data.